app/src/main/java/org/simple/clinic/drugs/PrescribedDrug.kt

```diff
@@ -19,19 +17,7 @@ import java.util.UUID
  * Drugs prescribed to a patient. This may have been picked
  * from a [ProtocolDrug] or entered manually.
  */
-@Entity(
-    foreignKeys = [
-      ForeignKey(
-          entity = Facility::class,
-          parentColumns = ["uuid"],
-          childColumns = ["facilityUuid"],
-          onDelete = ForeignKey.CASCADE,
-          onUpdate = ForeignKey.CASCADE)
-    ],
-    indices = [
-      Index("patientUuid", unique = false),
-      Index("facilityUuid", unique = false)
-    ])
+@Entity(indices = [Index("patientUuid", unique = false)])
```

Comment:
We might want to consider keeping the index on `facilityUuid` while dropping the foreign key, depending on how frequently we query the table based on this field.

Reply:
Same: We don't query prescriptions based on facilities right now, so should be fine.